### DISCUSSION TITLE
Unifying Shortcuts for Akira

### DISCUSSION THREAD

**Discussion by iclonethefirst** [akiraux/Akira#602] [Wannabe Documentation]
New users expect certain behavior and functionalities when they switch to an similar app. In order to keep track of the established conventions, I started this table to get a better overview of the current conventions and features.

I will add new items over time. If a function is missing, please write them down as markdown. 

# Legend:

Description | Symbol | Code
------------ | ------------- | -------------
Implemented|	:white_check_mark: | `:white_check_mark:`
Implemented, but different from the conventions  | :vs: | `:vs:`
In progress | :hourglass_flowing_sand: | `:hourglass_flowing_sand:`
Missing | :x: | `:x:`

# Shortcut status and comparison: 

Status | Issue |Akira | Functions | Figma | Adobe XD | Affinity Suite | Sketch |  
------------ | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | -------------
 :vs: |  |<kbd>shift</kbd> | resize from middle (mouse drag) | <kbd>alt</kbd> | <kbd>alt</kbd> | <kbd>ctrl</kbd> |<kbd>option</kbd> ( <kbd>alt</kbd> )  | 
 :hourglass_flowing_sand: |  |<kbd>shift</kbd> | select multiple items (click) | <kbd></kbd> | <kbd></kbd> | <kbd></kbd> |<kbd></kbd> ( <kbd></kbd> )  |

**Comment by ghost** [comment]
Use <kbd>shift</kbd> to resize with item's ratio

**Reply by Justin-Bento** [reply]
I would use [Sketch Shortcuts](https://www.sketch.com/docs/designing/shortcuts/) as inspiration due to them being straightforward and easy to remember.

**Comment by albfan** [comment]
At some point probably configurable shortcuts can be addressed. But probably an akira shorcuts will be always there as we want to suggest our own vision of "speed up and ease design"
